src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php

```diff
@@ -54,7 +54,7 @@ abstract class AbstractLanguageHandler extends AbstractObjectModelHandler
     protected function copyNoPictureImage(IsoCode $isoCode, $noPictureImagePath)
     {
         if (!($temporaryImage = tempnam(_PS_TMP_IMG_DIR_, 'PS'))
-            || !move_uploaded_file($noPictureImagePath, $temporaryImage)
+            || !copy($noPictureImagePath, $temporaryImage)
         ) {
```

Comment:
So the reason for this change, is that if something goes wrong in this function before file is moved, exception will be thrown(see second issue in https://github.com/PrestaShop/PrestaShop/pull/21878#issuecomment-756065766). Reason for the exception is that something (I guess symfony form) expects a file to still exist, but this function moves it to temp file location.

So by copying it instead of moving and deleting if this function works out correctly solves the issue, but I am not sure if this is best solution.